docs/UserGuide.md

```diff
@@ -417,9 +423,9 @@ before and after 2021-03-31 18:00
 
 #### 4.2.5 Fees
 
-##### Getting monthly fee for a particular student: `fee`
+##### Calculating monthly fee for a particular student: `fee`
```

Comment:
Actually not sure if we should use 'calculating', maybe 'display'?

Reply:
O.o I tink that aileen has mention that calculates in this case is better for the user to understand what the feature is doing. I will just keep it this way if you are ok with this.